exercises/two-bucket/bonus_test.go

```diff
@@ -13,6 +13,9 @@ var noSolutionCases = []bucketTestCase{
 		"No solution case 2",
 		3, 6, 1, "one", "", 0, 0, true,
 	},
+	{	"Adding Complexity",
```

Comment:
travis-ci failed for a "go fmt".  Can be fixed with:
```
-       {       "Adding Complexity",
+       {
+               "Adding Complexity",
```

Also suggest changing var noSolutionCases ... to var bonusCases ...,  since a solution can be found for the new test case.

Reply:
- checked with go fmt
- added the new variable and the new test case label